src/components/fpl/FantasyPlanner.tsx

```
@@ -0,0 +1,1677 @@
+                <p className="text-gray-500">
+                  Plan future transfers, chips, subs etc.
+                </p>
+              </div>
+              <div className="text-right">
+                <p
+                  className={`text-sm ${
+                    theme === "dark" ? "text-gray-300" : "text-gray-600"
+                  } cursor-pointer hover:text-blue-500"`}
+                  onClick={() => setShowManagerIdModal(true)}
+                >
+                  ID {currentManagerId} - Click to change
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)